tests/OmniSharp.Roslyn.CSharp.Tests/CodeActionsWithOptionsFacts.cs

```diff
@@ -15,6 +15,63 @@ public CodeActionsWithOptionsFacts(ITestOutputHelper output)
         {
         }
 
+        [Fact]
+        public async Task Can_extract_base_class()
+        {
+            const string code =
+                @"public class Class1[||]
+                {
+                    public string Property { get; set; }
+
+                    public void Method() { }
+                }";
+            const string expected =
+                @"public class NewBaseType
+                {
+                    public string Property { get; set; }
+
+                    public void Method() { }
+                }
+
+                public class Class1 : NewBaseType
+                {
+                }";
+
+            // TODO: this refactoring was renamed to 'Extract base class...' in latest Roslyn
```

Comment:
Thanks for documenting this!

Reply:
for the next lucky winner that updates Roslyn 😀